scripts/pre_commit_linter.py

```diff
@@ -323,14 +309,28 @@ def _pre_commit_linter():
     py_files_to_lint = [
         filename for filename in all_files if filename.endswith('.py')]
 
-    summary_messages = []
-    print '\nStarting jscs linter...'
-    summary_messages.append(_lint_js_files(
-        node_path, jscs_path, config_jscsrc, js_files_to_lint))
-    print '\nStarting pylint...'
-    summary_messages.append(_lint_py_files(
-        config_pylint, py_files_to_lint))
+    results = multiprocessing.Queue()
```

Comment:
suggest breaking into js_lint_results and python_lint_results; then, when ordering output, print python_lint_results followed by js_lint_results (to match the order shown in the terminal)